apps/nxls/src/main.ts

```diff
@@ -109,6 +109,8 @@ import { URI } from 'vscode-uri';
 import { ensureOnlyJsonRpcStdout } from './ensureOnlyJsonRpcStdout';
 import { NativeWatcher } from '@nx-console/shared-watcher';
 
+const connection = createConnection(ProposedFeatures.all);
```

Comment:
this is a convenience - it shows a better error message when you run `nxls` without any arguments (as the error message says "use --stdio or one of the other modes"

Reply:
that makes sense, an error during connection creation is one of the few ones we don't want to suppress